File: Entity/Entities/Hijo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entity.Entities
{
    public class Hijo
    {
        public int IdDerHab { get; set; }
        public string ApPaterno { get; set; }
        public string ApMaterno { get; set; }
        public string Nombre1 { get; set; }
        public string Nombre2 { get; set; }
        public string NombreCompleto { get; set; }
        public DateTime FchNac { get; set; }
        public bool Estado { get; set; }
        public Personal Padre { get; set; }
    }
}

File: LuckyJobs/ClientApp/src/app/models/Hijo.ts
import { Personal } from "./Personal";

export interface Hijo {
  idDerHab: number,
  apPaterno: string,
  apMaterno: string,
  nombre1: string,
  nombre2: string | null,
  nombreCompleto: string,
  fchNac: Date,
  padre: Personal,
  estado: boolean
}
